tests/Registry/RegistryTest.php

```diff
@@ -6,7 +6,6 @@
  * @subpackage Tests
  *
  * @link https://github.com/utopia-php/framework
- * @author Eldad Fux <eldad@appwrite.io>
```

Comment:
Let's remove the whole comment block, please.

Reply:
Done

Reply:
@Meldiron can you review it once